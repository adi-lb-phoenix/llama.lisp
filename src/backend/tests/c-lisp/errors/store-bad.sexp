(c-lisp
    (define ((main void))
        (declare a (ptr int))
        (set a (alloc int 3))
        (store a 5 6 7)))
