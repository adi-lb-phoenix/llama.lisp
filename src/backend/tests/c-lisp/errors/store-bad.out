Error in statement: ['store', 'a', 5, 6, 7]
CodegenError: Bad store expression: ['store', 'a', 5, 6, 7]
